godel_msgs/msg/ScanPlanParameters.msg

```diff
@@ -0,0 +1,12 @@
+# Path Parameters
+float64 scan_width           	# (m) actual width of scan tool
+float64 margin                  # (m) additional distance to leave near surface boundaries
+float64 overlap                 # (m) overlap distance between adjacent paths
+float64 approach_distance		# (m) approach distance
+float64 traverse_spd            # (m/s) Speed to travel while at traverse height
```

Comment:
To match the BlendingPlanParameters message which has similar discrepancies in its abbreviations. There are a number of issues with these messages, but I would strongly urge that we let it go for the moment, and log an issue.